lib/image-inspector-client.rb

```diff
@@ -74,13 +75,10 @@ def http_options
     end
 
     def http_headers
-      if @auth_options[:bearer_token]
-        {
-          Authorization: "Bearer #{@auth_options[:bearer_token]}"
-        }
-      else
-        {}
-      end
+      headers = {}
+      headers[:Authorization] = "Bearer #{@auth_options[:bearer_token]}" if @auth_options[:bearer_token]
+      headers[:'X-Auth-Token'] = @auth_options[:auth_token] if @auth_options[:auth_token]
```

Comment:
Since this looks quite custom I'm thinking maybe we should have a general headers param and then we can pass in what we like. Don't change yet. I assume we will need a similar change in kubeclient? I discussed this with @abonas. We need to understand better the reason for this custom auth header